.github/ISSUE_TEMPLATE/1_bug_report.yaml

```diff
@@ -86,6 +86,13 @@ body:
     validations:
       required: true
 
+  - type: markdown
+    attributes:
+      value: >
+        TIP:
+
+        Use this [script to gather](https://github.com/Lightning-AI/lightning/blob/master/requirements/collect_env_details.py) the **Important info**.
```

Comment:
Why isn't this part of the section above? Now we are requesting to gather this information twice.

Reply:
We aren't asking to gather the info twice, we're telling them they can use the script to gather the info. The reason it' at the bottom is because this is not WYSIWYG. The form is broken into sections. Some sections support MD while some only text in the description. I could move this TIP above the Important info section, but I don't think it can be directly included under the Important info heading.

Reply:
My point is that running this script will include the info for

```
        #- PyTorch Lightning Version (e.g., 1.5.0):
        #- Lightning App Version (e.g., 0.5.2):
        #- PyTorch Version (e.g., 1.10):
        #- Python version (e.g., 3.9):
        #- OS (e.g., Linux):
        #- CUDA/cuDNN version:
        #- Running environment of LightningApp (e.g. local, cloud):
```

So if the user manually fills it out AND runs the script, the info is gathered twice for those fields

Reply:
The field itself is not huge. The reader will see the tip at the bottom of the field. Let's merge it and see what it looks like. If it's sub-optimal at the bottom I'll move it above the Important info field.